utils/build-script

```diff
@@ -740,6 +780,23 @@ the number of parallel build jobs to use""",
                 "--symbols-package.")
             return 1
 
+    if args.android:
+        if args.android_ndk is None or \
+                args.android_ndk_version is None or \
+                args.android_icu_uc is None or \
+                args.android_icu_uc_include is None or \
+                args.android_icu_i18n is None or \
+                args.android_icu_i18n_include is None:
+            print_with_argv0("When building for Android, --android-ndk, "
+                             "--android-ndk-version, --android-icu-uc, "
+                             "--android-icu-uc-include, --android-icu-i18n, "
+                             "and --android-icu-i18n-include must be "
+                             "specified.")
+            return 1
+        # lib/Driver/ToolChains.cpp depends on this environment variable being
```

Comment:
I feel like we had this discussion before, but I don't see this variable being consumed by clang.  What does it do?  Why do we need to differ?